README.md

```diff
@@ -96,6 +96,8 @@ uint256 public constant tokensPerEth = 100;
 
 Edit `packages/hardhat/deploy/01_deploy_vendor.js` to deploy the `Vendor` (uncomment Vendor deploy lines).
 
+Uncomment the `Buy Tokens` sections in `packages\nextjs\pages\token-vendor.tsx` to show the UI to buy tokens on the Token Vendor tab.
```

Comment:
Should we also tell them to uncomment `tokenPerEth`? Or let them figure it out?

I mean both is fine for me and makes sense 🙌 just making sure we didn't forget to mention about it

Reply:
I think they should be able to figure it out.  Once the buyTokens section is uncommented, it will give them a nice error on tokensPerEth.  Let's let it ride like this and see how they handle it.